agent-runner-common/src/main/kotlin/com/epam/drill/agent/runner/AgentLoader.kt

```diff
@@ -1,18 +1,3 @@
-/**
```

Comment:
gh action is failing for both files on license-check stage

Reply:
done